egs2/mediaspeech/asr1/README.md

```diff
@@ -0,0 +1,31 @@
+<!-- Generated by scripts/utils/show_asr_result.sh -->
+# RESULTS
+## Environments
+- date: `Tue Mar 22 13:50:31 UTC 2022`
+- python version: `3.7.11 (default, Jul 27 2021, 14:32:16)  [GCC 7.5.0]`
+- espnet version: `espnet 0.10.7a1`
+- pytorch version: `pytorch 1.10.1`
+- Git hash: `1991a25855821b8b61d775681aa0cdfd6161bbc8`
+  - Commit date: `Mon Mar 21 22:19:19 2022 +0800`
+
```

Comment:
Please upload a model to the HF hub and add a link here.

Reply:
fixed